config.defaults.yaml

```diff
@@ -51,3 +51,7 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
       title: RR Lyrae c
+    delta_scuti:
+      coordinates: [271.6835652500, 30.0281826000]
```

Comment:
drop trailing zeros?

Reply:
done.

Reply:
I would actually round it off to 5 decimal places.
The period too should have a limited number of digits.